books/free-programming-books-langs.md

```diff
@@ -1899,6 +1901,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Natural Language Processing with Python](http://www.nltk.org/book/) (3.x)
 * [Non-Programmer's Tutorial for Python 3](https://en.wikibooks.org/wiki/Non-Programmer%27s_Tutorial_for_Python_3) - Wikibooks (3.3)
   * [Non-Programmer's Tutorial for Python 2.6](https://en.wikibooks.org/wiki/Non-Programmer%27s_Tutorial_for_Python_2.6) - Wikibooks (2.6)
+* [Obey the Testing Goat!](https://www.obeythetestinggoat.com/pages/book.html) - Harry Percival (HTML, PDF, Exercises sources)
```

Comment:
Removing patch to complete https://github.com/EbookFoundation/free-programming-books/pull/6496#discussion_r739629120 suggestion
```suggestion
```